j5basic/DictUtils.py

```diff
@@ -25,7 +32,9 @@ def unique_items(list1):
 
 def assert_dicts_equal(dict1,dict2, datetimes_to_naive=False):
     """tests equality of two dicts"""
-    k1, k2 = set(dict1.keys()), set(dict2.keys())
+    k1, k2 = list(dict1.keys()), list(dict2.keys())
```

Comment:
I didn't "change this" this is how it was. It got changed during the Python 3 update to be a `set` instead of a sorted list. I was a bit skeptical of this change since `set`s are not sorted in python, but I can redo the change if you want.

Reply:
oh, ok... so this is actually just a revert back to the original?

Reply:
Yep.